src/amber/cli/templates/app/README.md.ecr

```diff
@@ -1,59 +1,34 @@
 # <%= @name %>
 
-<%= @name %> is a new project using [Amber Framework](https://amberframework.org/)
+This is a project written using [Amber Framework](https://amberframework.org/).
 
 ## Installation
 
-Install shard dependencies.
-
-```
-shards install
-```
-
-Configure the `config/environments/development.yml` and set the `database_url` with your credentials to your database.
-
-Then:
-
-```
-amber db create migrate
-```
+1. [Install required dependencies](https://github.com/amberframework/online-docs/blob/master/getting-started/quick-start.md#install-crystal-and-amber)
+2. Run `shards install`
 
 ## Usage
 
-### Development
-
-To run amber development server:
+This will run a development server:
```

Comment:
I would advise keeping the "To do X, run Y" form of sentences, unless you want to change the style in the whole document uniformly. Also it could be good to keep the number of sentences which begin with "This" and similar words to a minimum.

I would suggest saying something more verbose, but more correct language-wise, such as:

"To run Amber [server] in development environment:"